values.yaml

```diff
@@ -371,6 +371,10 @@ server:
   # by setting the `server.extraConfig` value.
   connect: true
 
+  serviceAccount:
+    # A map of annotations to add to the server service account 
+    annotations: {}
```

Comment:
@cooperbenson-qz Are you able to format them as multiline strings for consistency with https://github.com/hashicorp/consul-helm/blob/master/values.yaml#L584?

Reply:
Oh I missed that was the pattern already in use. I'm used to having a map that uses `toYaml`, but I can use the established pattern